src/components/computed-properties.vue

```diff
@@ -259,18 +274,21 @@ export default {
       this.emptyForm();
       this.displayList = false;
     },
-    validateDuplicate() {
-      this.existsProperty = false;
-      this.current.forEach(item => {
-        if (item.property === this.add.property && item.id !== this.add.id) {
-          this.existsProperty = true;
+    validateData() {
+      this.ruleDescription = 'required';
+      this.ruleFormula = 'required';
+      this.rulePropName = this.rulePropName ? this.rulePropName : 'required';
+      this.editorInvalid = this.add.formula.trim() === '';
+      let fields = this.$children[0].$children[0].$children;
+
+      let valid = false;
```

Comment:
Since you end up treating this variable as a number, I think it would make more sense to initialize it to `0` instead of `false`.

Reply:
changed